livelossplot/main_logger.py

```diff
@@ -158,3 +161,9 @@ def log_history(self, value: Dict[str, List[LogItem]]) -> None:
         if len(value) > 0:
             raise RuntimeError('Cannot overwrite log history with non empty dictionary')
         self._log_history = value
+
+    def step_name(self, group_name: str) -> str:
+        if isinstance(self._step_names, str):
+            return self._step_names
+        else:
+            return self._step_names.get(group_name, 'epoch')
```

Comment:
It is OK, but I think it would be more idiomatic to use DefaultDict (but it can stay as it is).